test/e2e/kubectl.go

```diff
@@ -164,6 +164,17 @@ var _ = Describe("Kubectl client", func() {
 				Failf("Unexpected kubectl exec output. Wanted '%s', got '%s'", execOutput, expectedExecOutput)
 			}
 		})
+
+		It("should support inline execution and attach", func() {
+			By("executing a command with run and attach")
+			runOutput := runKubectl("run", fmt.Sprintf("--namespace=%v", ns), "run", "run-test", "--image=busybox", "--restart=Never", "--attach", "echo", "running", "in", "container")
+			expectedRunOutput := "running in container"
+			if runOutput != expectedRunOutput {
```

Comment:
this is fine, but the more idiomatic Ginkgo (and succinct) way to write this would be
`Expect(runOutput).To(Equal("running in container"), "kubectl run --attach")`

or something like that